apps/backend/src/lib/controller/oauth/bespoke/token/index.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  GITHUB,
+  OAuthMapping,
+  STRAVA
+} from "@types";
+
+export async function stravaFetchToken(mapping: OAuthMapping, code: string): Promise<globalThis.Response> {
```

Comment:
what is this used for? i thought for strava we are going to do client side fetching

Reply:
I kept it there just in case we ever needed to do server-side fetching but the point is well taken that (1) we should default to client-side fetching / discourage server-side fetching and (2) it's confusing to have this here unused.